src/include/gurt/telemetry_common.h

```diff
@@ -18,6 +18,43 @@
 #define D_TM_SHARED_MEMORY_KEY		0x10242048
 #define D_TM_SHARED_MEMORY_SIZE		(1024 * 1024)
 
+#define D_TM_MIN(stats, name) \
+	stats->dtm_min.min_##name
+
+#define D_TM_MAX(stats, name) \
+	stats->dtm_max.max_##name
+
+#define D_TM_PRINT_STATS(s, stats, name, fmt) \
+	fprintf(s, " min: %"#fmt " max: %"#fmt " mean: %lf size: %" PRIu64, \
+		D_TM_MIN(stats, name), D_TM_MAX(stats, name),               \
+		stats->mean, stats->sample_size);                           \
+	if (stats->sample_size > 2)                                         \
+		fprintf(s, " std dev: %lf", stats->std_dev);
+
+#define D_TM_COMPUTE_MIN(value, node, name)                                 \
+	if ((value) < node->dtn_metric->dtm_stats->dtm_min.min_##name)      \
+		node->dtn_metric->dtm_stats->dtm_min.min_##name = (value);
+
+#define D_TM_COMPUTE_MAX(value, node, name)                                 \
+	if ((value) > node->dtn_metric->dtm_stats->dtm_max.max_##name)      \
+		node->dtn_metric->dtm_stats->dtm_max.max_##name = (value);
+
+#define D_TM_COMPUTE_MEAN(node, value)                                      \
+	node->dtn_metric->dtm_stats->mean +                                 \
+		(((value) - node->dtn_metric->dtm_stats->mean) /            \
+		node->dtn_metric->dtm_stats->sample_size);
+
+#define D_TM_VALUE(node)                                                    \
+	(node->dtn_type & D_TM_DURATION) ?                                  \
+		(node->dtn_metric->dtm_data.tms[0].tv_sec +                 \
+			(node->dtn_metric->dtm_data.tms[0].tv_nsec / 1E9)) :\
+		(node->dtn_metric->dtm_data.value)
+
+#define D_TM_COMPUTE_SUM_OF_SQUARES(node, value)                            \
```

Comment:
(style)  Macros with multiple statements should be enclosed in a do - while loop
(style)  macros should not use a trailing semicolon